backup/queries_table_defs.go

```diff
@@ -664,7 +664,7 @@ func GetTableInheritance(connectionPool *dbconn.DBConn, tables []Relation) map[u
 		}
 		// If we are filtering on tables, we only want to record dependencies on other tables in the list
 		if len(tableOidList) > 0 {
-			tableFilterStr = fmt.Sprintf("\nAND i.inhrelid IN (%s)", strings.Join(tableOidList, ","))
+			tableFilterStr = fmt.Sprintf("WHERE i.inhrelid IN (%s)", strings.Join(tableOidList, ","))
```

Comment:
Are you sure it is good idea to put `where` in condition variable?

Reply:
Since we have no other conditions, WHERE is needed only in the case of this filter.